File: C++/cnt_zeroes.cpp
#include <iostream>

int cnt_zeroes(int a[], int n){
    int cnt = 0;
    for (int i = 0; i < n; i++)
    {
        if (a[i] == 0)
        {
            cnt ++;
        }
    }
        return cnt;
    }
    

File: Methods/Methods-Java/Java source-code/cnt_zeroes.java


public class cnt_zeroes {
    public static int cnt_zeroes_m(Integer[] a) {
        int cnt = 0;
        for (int i = 0; i < a.length; i++) {
            if (a[i] == 0) {
                cnt++;
            }
        }
        return cnt;
    }
}